src/qt/bitcoingui.cpp

```diff
@@ -412,7 +412,6 @@ void BitcoinGUI::createActions()
 #endif // ENABLE_WALLET
 
     connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_C), this), &QShortcut::activated, this, &BitcoinGUI::showDebugWindowActivateConsole);
-    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::SHIFT + Qt::Key_D), this), &QShortcut::activated, this, &BitcoinGUI::showDebugWindow);
```

Comment:
Removing this shortcut is not mentioned in the PR description.
Moreover, I'd prefer to keep it as is.

Reply:
Oh, my bad, the change was in response to an earlier review by @promag.

Just read the refs and reverted the change :+1: